launcher/src/backend/ethereum-services/LidoObolExitService.js

```diff
@@ -0,0 +1,57 @@
+import { NodeService } from "./NodeService";
+import { ServiceVolume } from "./ServiceVolume";
+
+export class LidoObolExitService extends NodeService {
+  static buildByUserInput(network, ports, dir) {
+    const service = new LidoObolExitService();
+    service.setId();
+    const workingDir = service.buildWorkingDir(dir);
+
+    const image = "obolnetwork/lido-dv-exit";
+
+    const exitmessagesDir = "/exitmessages";
+    const charonDir = "/charon";
+
+    const volumes = [
+      new ServiceVolume(workingDir + "/exitmessages", exitmessagesDir),
+      new ServiceVolume(workingDir + "/charon", charonDir),
+    ];
+
+    service.init(
+      "LidoObolExitService", //service
+      service.id, // id
+      1, // configVersion
+      image, // image
+      "latest", // imageVersion
+      [
+        "run",
+        "--beacon-node-url=http://lighthouse:5052",
```

Comment:
only Lighthouse?

Reply:
this is just a placeholder for the beacon service

Reply:
since its just placeholder. imho, to use something like `http://beaconNode:beaconPort` is better, isn't it?

Reply:
sure, but as i'm the only one using it is just went for the quick and dirty sollution

Reply:
got it 😄